Jenkinsfile

```diff
@@ -39,6 +39,11 @@ String stan_pr() {
     }
 }
 
+def isBranch(String b) { env.BRANCH_NAME == b }
+Boolean isPR() { env.CHANGE_URL != null }
+String fork() { env.CHANGE_FORK ?: "stan-dev" }
+String branchName() { isPR() ? env.CHANGE_BRANCH :env.BRANCH_NAME }
```

Comment:
These were missing from the Jenkinsfile which is why the check was failing. @serban-nicusor-toptal can you look this over at your convenience and lmk if it looks alright

Reply:
I see, so there was actually a function.
The strange thing about this is that I ran it through Jenkins own Pipeline Linter and Checker and it didn't throw anything -.-